app/src/main/res/layout/item_overdue_list.xml

```diff
@@ -6,65 +6,126 @@
   android:layout_height="wrap_content"
   android:layout_marginBottom="@dimen/spacing_8">
 
-  <LinearLayout
+  <RelativeLayout
     android:layout_width="match_parent"
-    android:layout_height="wrap_content"
-    android:orientation="horizontal">
+    android:layout_height="wrap_content">
 
     <LinearLayout
-      android:layout_width="0dp"
+      android:id="@+id/patient_details"
+      android:layout_width="match_parent"
       android:layout_height="wrap_content"
-      android:layout_weight="1"
+      android:orientation="horizontal">
+
+      <LinearLayout
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_weight="1"
+        android:orientation="vertical"
+        android:padding="@dimen/spacing_16">
+
+        <TextView
+          android:id="@+id/overdue_patient_name_gender_age"
+          android:layout_width="wrap_content"
+          android:layout_height="wrap_content"
+          android:layout_marginBottom="@dimen/spacing_4"
+          android:textColor="@color/black_opacity_90"
+          android:textSize="@dimen/textsize_16"
+          tools:targetApi="lollipop"
+          tools:text="Anisha Acharya" />
+
+        <TextView
+          android:id="@+id/overdue_patient_bp"
+          android:layout_width="wrap_content"
+          android:layout_height="wrap_content"
+          android:layout_marginBottom="@dimen/spacing_4"
+          android:textColor="@color/black_opacity_50"
+          android:textSize="@dimen/textsize_14"
+          tools:text="30 days ago: 175/85" />
+
+        <TextView
+          android:id="@+id/overdue_days"
+          android:layout_width="wrap_content"
+          android:layout_height="wrap_content"
+          android:textColor="@color/red_500"
+          android:textSize="@dimen/textsize_14"
+          tools:text="4 day overdue" />
+      </LinearLayout>
+
+      <View
+        android:id="@+id/overdue_separator"
+        android:layout_width="1dp"
+        android:layout_height="match_parent"
+        android:layout_marginBottom="@dimen/spacing_8"
+        android:layout_marginTop="@dimen/spacing_8"
+        android:background="@color/black_opacity_10" />
+
+      <ImageButton
+        android:id="@+id/overdue_patient_call"
+        android:layout_width="wrap_content"
+        android:layout_height="match_parent"
+        android:layout_gravity="center_vertical|end"
+        android:background="?android:attr/selectableItemBackground"
+        android:paddingEnd="@dimen/spacing_16"
+        android:paddingStart="@dimen/spacing_16"
+        android:src="@drawable/ic_call"
```

Comment:
Can we rename `ic_call` to include its size? That's how we've named all our icons so far.